x509/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![no_std]
+#![cfg_attr(not(test), no_std)]
```

Comment:
Was there a specific reason for this? I couldn't see why it was needed.

The main problem with doing this is potential build failures for `std`-dependent (or `std` prelude-dependent) features that you only see when doing `cargo build` that are masked when doing `cargo test`

Reply:
I did it because tests didn't run without it. But I forget why. I can investigate this afternoon.